.github/workflows/evolution.yml

```
@@ -5,9 +5,10 @@ name: CI
 
 on:
   push:
-    branches: [ main, od_2023, v0.3.0 ]
+    branches: [ main, od_2023, v0.3.0, v0.5.0 ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)